ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -225,10 +213,12 @@ public void handlePost(RestRequest restRequest, RestResponseChannel restResponse
     long processingStartTime = System.currentTimeMillis();
     long preProcessingTime = 0;
     handlePrechecks(restRequest, restResponseChannel);
-    restRequest.getMetricsTracker().injectMetrics(frontendMetrics.postBlobMetrics);
-    RestRequestMetrics requestMetrics =
-        restRequest.getSSLSession() != null ? frontendMetrics.postBlobSSLMetrics : frontendMetrics.postBlobMetrics;
-    restRequest.getMetricsTracker().injectMetrics(requestMetrics);
+    boolean isSsl = restRequest.getSSLSession() != null;
```

Comment:
isn't it cleaner the old way (ternary operator)? just delete line 228 (which looks redundant)